src/__tests__/tigris.rpc.spec.ts

```diff
@@ -370,9 +370,11 @@ describe("rpc tests", () => {
 		const tigris = new Tigris({ serverUrl: "localhost:" + SERVER_PORT, projectName: "db3" });
 		const db1 = tigris.getDatabase();
 		const readOnePromise = db1.getCollection<IBook>("books").findOne({
-			op: SelectorFilterOperator.EQ,
-			fields: {
-				id: 1,
+			filter: {
+				op: SelectorFilterOperator.EQ,
```

Comment:
does it need to be explicit?

Reply:
I'll add simpler examples in next iteration